tests/test_cross_validation.cc

```diff
@@ -169,6 +169,26 @@ TEST(test_crossvalidation, test_leave_one_out_conditional_variance) {
   EXPECT_LE((loo_marginal.covariance.diagonal() - loo_variance).norm(), 1e-8);
 }
 
+TEST(test_crossvalidation, test_leave_one_out_conditional) {
```

Comment:
Do you want to test that you see an outlier assigned appropriate low likelihood when left out?